### ISSUE TITLE
Datagrid placeholder always visible when using pagination

### ISSUE THREAD

**Issue by shad1w** [vmware/clarity#2391]
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

When using a datagrid with pagination, the bottom border of the last item should align with the datagrid footer.

### Actual behavior

An empty placeholder (div with class `datagrid-placeholder`) is shown between the last entry of the datagrid and the footer.
![empty placeholder is shown](https://user-images.githubusercontent.com/22195087/41809195-8a5081ea-76e9-11e8-8fb5-a40dc0a64b73.png)

### Reproduction of behavior

https://stackblitz.com/edit/datagrid-placeholder-bug

### Environment details

* **Angular and Clarity version:** As in https://stackblitz.com/edit/clarity-light-theme-v11

* **OS and version:** Windows 10

* **Browser:** Likely all (tested with Chrome 67, Firefox 60 and MS Edge 41 with EdgeHTML 16)

**Comment by hippee-lee** [comment]
Thanks for reporting this. It does look like the datagrid height is not being calculated correctly and it should also be constrained to a container (see a modification of this StackBlitz fork: https://stackblitz.com/edit/datagrid-placeholder-bug-qu7btz)

**Comment by DDDGamer** [comment]
I was able to fix by adding
```
.datagrid-host {
    height: auto!important;
}
```

**Comment by youdz** [comment]
Fixed by #2670.
